negotiation/src/test/java/it/eng/negotiation/model/ModelUtil.java

```diff
@@ -13,8 +13,8 @@ public class ModelUtil {
 	public static final String CALLBACK_ADDRESS = "https://callback.address/callback";
 	public static final String FORWARD_TO = "https://forward-to.com";
 	public static final String DATASET_ID = "urn:uuid:DATASET_ID";
-	public static final String ASSIGNEE = "urn:uuid:ASSIGNEE";
-	public static final String ASSIGNER = "urn:uuid:ASSIGNER";
+	public static final String ASSIGNEE = "ASSIGNEE_CONSUMER";
```

Comment:
Can we leave this prefix urn:uuid:, so that we know that in real life scenario this will be some sort of uuid?